test/Npgsql.Tests/TestUtil.cs

```diff
@@ -534,3 +534,5 @@ public enum PooledOrNot
     Pooled,
     Unpooled
 }
+
+enum Mood { Sad, Ok, Happy }
```

Comment:
I'd rather we kept this in the specific test classes rather than making it available in all classes, even if it's a bit of duplication...

Reply:
Agreed